news/2023/2023-10-22-halloween-descends.md

```diff
@@ -0,0 +1,121 @@
+
+Decoration is the name of the game in [**Triantafyllia**](https://osu.ppy.sh/users/17084594)'s piece, impressing **1122** voters. So much going on here between pippi and Mocha's incredibly ornate costumes and vibrant artistry really gunning for a beautiful collision of candy-like light and colour.
+
+[![](https://assets.ppy.sh/contests/186/banners/AyoLie.jpg)](https://assets.ppy.sh/contests/186/winners/AyoLie.png)
+
+UHHHHHH BRANES. [**AyoLie**](https://osu.ppy.sh/users/29186645) DREW ZOMBIE PIPPI. ZOMBIE MOCHA TOO. UUUURRRRRR. **1115** VOTERS LIKE VERY MUCH. BRAAAAANES. Also, a cute Chirou's chilling too. We're not entirely sure if she's a disembodied head or not, but we're going to pretend she is, because you know, Halloween. Wonderful piece!
+
+[![](https://assets.ppy.sh/contests/186/banners/Haru34_old.jpg)](https://assets.ppy.sh/contests/186/winners/Haru34_old.png)
+
+**1079** voter ghosts are waiting patiently in line to be hugged by [**Haru34_old**](https://osu.ppy.sh/users/8806199)'s pippi-mummy amidst a spooky violet-tinged moonscape. Why are these spirits so unbearably cute? It is a mystery.
+
+## peppy's Picks
+
+Long ago before time was counted, the Dean "peppy" Herbert once uttered deep within the hallowed crypts (of our internal Discord server) that there were too many good entries and not enough votes to go around. For thousands of years (approximately eleven seconds, we counted) did he contemplate, before deciding that he would pick a selection of his favorite entries to join our finalists on the main menu anyway. We've since discovered that if you gave him triple the number of ordinary votes, he'd still run out of them somehow.
+
+Thus, the Picks were born. And now, we all benefit.
+
+[![](https://assets.ppy.sh/contests/186/banners/nhyogu.jpg)](https://assets.ppy.sh/contests/186/winners/nhyogu.png)
+
+Normal Chirou. Orange Chirou. Evil Chirou. [**nhyogu**](https://osu.ppy.sh/users/7430439) Chirou. Orange Chirou. Evil Chirou. Orange Chirou. **EVIL CHIROU.**
+
+[![](https://assets.ppy.sh/contests/186/banners/Christi_Anna.jpg)](https://assets.ppy.sh/contests/186/winners/Christi_Anna.png)
+
+[**Christi_Anna**](https://osu.ppy.sh/users/11016739) gathered forth an assembly of mascots most stylish, including pippi, Chirou, Yuzu, the warring twins (Mani and Mari) plus Mocha and a don to enjoy some candelit Halloween festivities. Look at the cat!
+
+[![](https://assets.ppy.sh/contests/186/banners/Amythenia.jpg)](https://assets.ppy.sh/contests/186/winners/Amythenia.png)
+
+[**Amythenia**](https://osu.ppy.sh/users/19931859)'s witch pippi blazes along the latest experimental osu! mania key mode, which is so long that it quite literally comes from the moon. We did the math, and it'd be about *20.666 billion keys*, assuming the average width of a piano key. That's a lot of keys, almost enough to match the amount of cute going on in this piece. There's a pumpkin don too!
```

Comment:
https://en.wikipedia.org/wiki/Musical_keyboard

> **Modern piano keyboards ordinarily have an octave span of 164–165 mm** (6.46–6.50 in), resulting in the width of black keys averaging 13.7 mm (0.54 in) and white keys about 23.5 mm (0.93 in) at the base, disregarding space between keys.

![image](https://github.com/ppy/osu-wiki/assets/36758269/f6578f7c-4d4a-4e0f-bf39-127f7346e044)

```suggestion
[**Amythenia**](https://osu.ppy.sh/users/19931859)'s witch pippi blazes along the latest experimental osu!mania key mode, which is so long that it quite literally comes from the moon. We did the math, and it'd be about *27 billion keys*, assuming the average width of a piano key. That's a lot of keys, almost enough to match the amount of cute going on in this piece. There's a pumpkin don too!
```

Reply:
calculated using average millimeter width for black and white keys (186.mm)

Reply:
ok i don't think we need to go this hard on a 3 word haha funny quibble, i'll change to 28bil